conan/internal/model/conf.py

```diff
@@ -133,6 +133,10 @@
     "tools.build:linker_scripts": "List of linker script files to pass to the linker used by different toolchains like CMakeToolchain, AutotoolsToolchain, and MesonToolchain",
     # Package ID composition
     "tools.info.package_id:confs": "List of existing configuration to be part of the package ID",
+    # Runners
+    "runner.type": "Type of runner to use. Possible values: docker, ssh",
+    "runner.host": "Hostname of the remote machine to connect to",
+    "runner.configfile": "(boolean/str) Enable the use of the SSH configuration file (False by default). If a string is provided, it will be used as the path to the SSH configuration file.",
```

Comment:
Please do not introduce new ``runner`` categories in ``conf``.

All conf is either ``core.xxx``, ``tools.xxx`` or ``core.xxx``. 

At this moment, lets limit runners configuration to profiles ``[runner]`` section.

Reply:
Okay, I'm agree with you, I'm removing it